business_requirement_deliverable/models/project.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    pricelist_id = fields.Many2one(
+        string='Pricelist',
```

Comment:
"Estimation Pricelist"
help: "Pricelist used for the estimation of the Business Requirements deliverable. Currency of the deliverable will be the one from this pricelist."